src/components/Books/AddBook.js

```diff
@@ -0,0 +1,24 @@
+import '../Styles/AddBook.css';
+
+const AddBook = () => (
+  <div className="form-add-book">
+    <h1 className="add-new-book">Add New Books</h1>
+    <form className="form-container">
+      <input
+        type="text"
+        className="input-text"
+        placeholder="Add Book..."
+      />
+      <select>
+        <option value="Action">Action</option>
+        <option value="Scinenc Fiction">Science Fiction</option>
+        <option value="Economy">Economy</option>
+      </select>
```

Comment:
- [ ] Good job on making the categories section in a well-structured way 👍 However, from the project requirements, there are supposed to be 2 inputs, one for the **Title** and another for the **Author**
![image](https://user-images.githubusercontent.com/90222110/209320020-f605600d-c868-4ce3-9bff-ea6ea4a4415d.png)